apps/wallet/src/ui/app/shared/text/index.tsx

```diff
@@ -58,6 +58,9 @@ const textStyles = cva([], {
             true: 'font-mono',
             false: 'font-sans',
         },
+        lineHeight: {
```

Comment:
looking for suggestion here. The lineheight is really crammed when text is wrapped